books/free-programming-books-langs.md

```diff
@@ -1469,6 +1469,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 ### MongoDB
 
+* [How To Manage a Redis Database](https://www.digitalocean.com/community/books/how-to-manage-a-redis-database-ebook) - Mark Drake (PDF, EPUB)
```

Comment:
It was well in NoSQL. In other listings with several resources there are a category apart.

Could you restore it into this category?